diff -uNr ctwm-3.5.2.orig/Imakefile ctwm-3.5.2/Imakefile
--- ctwm-3.5.2.orig/Imakefile	Mon Nov 24 11:19:39 1997
+++ ctwm-3.5.2/Imakefile	Tue Sep 25 22:23:14 2001
@@ -133,7 +133,7 @@
 AllTarget(ctwm)
 
 SpecialObjectRule(parse.o,parse.c,'-DSYSTEM_INIT_FILE="'$(DESTDIR)$(TWMDIR)'/system.ctwmrc"')
-SpecialObjectRule(ctwm.o,ctwm.c,'-DPIXMAP_DIRECTORY="'$(DESTDIR)$(TWMDIR)'/images"')
+SpecialObjectRule(ctwm.o,ctwm.c,'-DPIXMAP_DIRECTORY="'$(DESTDIR)$(LIBDIR)'/ctwm/images"')
 #if !HasPutenv
 SpecialObjectRule(util.o,util.c,-DNOPUTENV)
 #endif
@@ -145,10 +145,10 @@
 InstallNonExecFile(system.ctwmrc,$(TWMDIR))
 #ifdef XPM
 install:: xpm
-	MakeDir($(DESTDIR)$(TWMDIR)/images)
+	MakeDir($(DESTDIR)$(LIBDIR)/ctwm/images)
 	@case '${MFLAGS}' in *[i]*) set +e;; esac; \
 	for i in xpm/?* ; do \
-		(set -x; $(INSTALL) -c $(INSTLIBFLAGS) $$i $(DESTDIR)$(TWMDIR)/images); \
+		(set -x; $(INSTALL) -c $(INSTLIBFLAGS) $$i $(DESTDIR)$(LIBDIR)/ctwm/images); \
 	done
 #endif
 
diff -uNr ctwm-3.5.2.orig/Imakefile.X11R4 ctwm-3.5.2/Imakefile.X11R4
--- ctwm-3.5.2.orig/Imakefile.X11R4	Mon Nov 24 11:19:40 1997
+++ ctwm-3.5.2/Imakefile.X11R4	Tue Sep 25 22:23:55 2001
@@ -110,7 +110,7 @@
 AllTarget(ctwm)
 
 SpecialObjectRule(parse.o,parse.c,'-DSYSTEM_INIT_FILE="'$(TWMDIR)'/system.ctwmrc"')
-SpecialObjectRule(ctwm.o,ctwm.c,'-DPIXMAP_DIRECTORY="'$(TWMDIR)'/images"')
+SpecialObjectRule(ctwm.o,ctwm.c,'-DPIXMAP_DIRECTORY="'$(LIBDIR)'/ctwm/images"')
 #if !HasPutenv
 SpecialObjectRule(util.o,util.c,-DNOPUTENV)
 #endif
@@ -122,10 +122,10 @@
 InstallNonExecFile(system.ctwmrc,$(TWMDIR))
 #ifdef XPM
 install:: xpm
-	MakeDir($(DESTDIR)$(TWMDIR)/images)
+	MakeDir($(DESTDIR)$(LIBDIR)/ctwm/images)
 	@case '${MFLAGS}' in *[i]*) set +e;; esac; \
 	for i in xpm/?* ; do \
-		(set -x; $(INSTALL) $$i $(DESTDIR)$(TWMDIR)/images); \
+		(set -x; $(INSTALL) $$i $(DESTDIR)$(LIBDIR)/ctwm/images); \
 	done
 #endif
 
